[1/6] cd /home/varun/ESP_STM_FILE/OTA/build/esp-idf/esptool_py && /home/varun/.espressif/python_env/idf5.4_py3.10_env/bin/python /home/varun/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 partition --type app /home/varun/ESP_STM_FILE/OTA/build/partition_table/partition-table.bin /home/varun/ESP_STM_FILE/OTA/build/OTA_ESP.bin
OTA_ESP.bin binary size 0xce8c0 bytes. Smallest app partition is 0x1f0000 bytes. 0x121740 bytes (58%) free.
[2/6] cd /home/varun/ESP_STM_FILE/OTA/build && /home/varun/.espressif/python_env/idf5.4_py3.10_env/bin/python /home/varun/esp/esp-idf/components/spiffs/spiffsgen.py 0x100000 /home/varun/ESP_STM_FILE/OTA/spiffs /home/varun/ESP_STM_FILE/OTA/build/storage.bin --page-size=256 --obj-name-len=32 --meta-len=4 --use-magic --use-magic-len
[3/6] Performing build step for 'bootloader'
[1/1] cd /home/varun/ESP_STM_FILE/OTA/build/bootloader/esp-idf/esptool_py && /home/varun/.espressif/python_env/idf5.4_py3.10_env/bin/python /home/varun/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 bootloader 0x1000 /home/varun/ESP_STM_FILE/OTA/build/bootloader/bootloader.bin
Bootloader binary size 0x6570 bytes. 0xa90 bytes (9%) free.
[3/4] cd /home/varun/esp/esp-idf/components/esptool_py && /usr/bin/cmake -D IDF_PATH=/home/varun/esp/esp-idf -D "SERIAL_TOOL=/home/varun/.espressif/python_env/idf5.4_py3.10_env/bin/python;;/home/varun/esp/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/varun/ESP_STM_FILE/OTA/build -P /home/varun/esp/esp-idf/components/esptool_py/run_serial_tool.cmake
esptool.py --chip esp32 -p /dev/ttyUSB1 -b 460800 --before=default_reset --after=hard_reset write_flash --flash_mode dio --flash_freq 40m --flash_size 4MB 0x1000 bootloader/bootloader.bin 0x10000 OTA_ESP.bin 0x8000 partition_table/partition-table.bin 0x200000 storage.bin
esptool.py v4.9.0
Serial port /dev/ttyUSB1

A fatal error occurred: Could not open /dev/ttyUSB1, the port is busy or doesn't exist.
([Errno 16] could not open port /dev/ttyUSB1: [Errno 16] Device or resource busy: '/dev/ttyUSB1')

FAILED: CMakeFiles/flash /home/varun/ESP_STM_FILE/OTA/build/CMakeFiles/flash 
cd /home/varun/esp/esp-idf/components/esptool_py && /usr/bin/cmake -D IDF_PATH=/home/varun/esp/esp-idf -D "SERIAL_TOOL=/home/varun/.espressif/python_env/idf5.4_py3.10_env/bin/python;;/home/varun/esp/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/varun/ESP_STM_FILE/OTA/build -P /home/varun/esp/esp-idf/components/esptool_py/run_serial_tool.cmake
ninja: build stopped: subcommand failed.
